tests/fixtures/core_update_tests/core/core_api.yml

```diff
@@ -0,0 +1,4 @@
+location:
+  type: app_store
+  name: tk-core
+  version: v0.18.91
```

Comment:
I had to create a new fixture as non of the existing ones seemed to do what I wanted.